src/ChaostreffScheduler.hs

```diff
@@ -27,76 +27,64 @@ schedule = do
 scheduleChaostreffs :: (MonadIO m, MonadReader AppCfg m) => m ()
 scheduleChaostreffs = do
   template <- loadLektorEventTemplate "contents-chaostreff.tmpl"
-
-  dates <- chaostreffDates
-  let events = fmap (\d -> template { eventTitle = "Chaostreff", eventDate = d }) dates
-
-  mapM_ createLektorCalendarEntry events
+  days <- chaostreffDays
+  mapM_ (createLektorCalendarEntry $ template { eventTitle = "Chaostreff"}) days
```

Comment:
Currently, we update the `eventTitle` here, because it's always 'Template' in the template files.

Example:
https://github.com/section77/website/blob/master/templates/chaostreff-scheduler/contents-chaostreff.tmpl#L2-L4

When the title is correct in the template files, we could simply use `mapM_ (createLektorCalendarEntry template) days`